syntax = "proto3";

message User {
  int64 id = 1;
  string name = 2;
}

message CreateUserRequest {
  string name = 1;
}

message CreateUserResponse {
  User user = 1;
}

service UserService {
  rpc CreateUser (CreateUserRequest) returns (CreateUserResponse) {}
}
